src/memtable.c

```diff
@@ -307,15 +307,19 @@ memtable_context_create(platform_heap_id hid,
                         process_fn       process,
                         void            *process_ctxt)
 {
+   platform_memfrag  memfrag_ctxt = {0};
    memtable_context *ctxt =
       TYPED_FLEXIBLE_STRUCT_ZALLOC(hid, ctxt, mt, cfg->max_memtables);
-   ctxt->cc = cc;
+   ctxt->mf_size = memfrag_size(&memfrag_ctxt);
```

Comment:
And the 2nd pattern you will see is:

After a successful memory allocation, code will (and needs to):

1. Save off the size of the memory fragment allocated using `memfrag_size()`.
2. Consistently, all structures that need to track this fragment-size will have a field named `mf_size`. 
3. Much later, `platform_free()` will be invoked using this `structure->mf_size` field.

I have tried to keep this name the same in all places. If you find something is not named like so, flag it and I will fix that.